client/lib/analytics/ad-tracking/record-order.js

```diff
@@ -74,6 +76,7 @@ export async function recordOrder( cart, orderId ) {
 	recordOrderInBing( cart, orderId, wpcomJetpackCartInfo );
 	recordOrderInQuantcast( cart, orderId, wpcomJetpackCartInfo );
 	recordOrderInCriteo( cart, orderId );
+	recordOrderInGAEnhancedEcommerce( cart, orderId );
```

Comment:
Do we need (can we leverage) `wpcomJetpackCartInfo` here?

Reply:
Yes, thank you.